modular_citadel/code/datums/mood_events/moodular.dm

```diff
@@ -0,0 +1,41 @@
+// Modular stuff to use with Citadel-specific moods.
+
+// box of hugs
+/obj/item/storage/box/hug/attack_self(mob/user)
+	. = ..()
+	GET_COMPONENT_FROM(mood, /datum/component/mood, user)
+	if(mood)
+		mood.add_event("hugbox", /datum/mood_event/hugbox)
+
+// headpats (IMPORTANT)
+/mob/living/carbon/help_shake_act(mob/living/carbon/M)
+	. = ..()
+	GET_COMPONENT_FROM(mood, /datum/component/mood, src)
+	if(mood)
+		mood.add_event("headpat", /datum/mood_event/headpat)
+
+// plush petting
+/obj/item/toy/plush/attack_self(mob/user)
+	. = ..()
+	if(stuffed || grenade)
+		GET_COMPONENT_FROM(mood, /datum/component/mood, user)
+		if(mood)
+			mood.add_event("plushpet", /datum/mood_event/plushpet)
+	else
+		GET_COMPONENT_FROM(mood, /datum/component/mood, user)
+		if(mood)
+			mood.add_event("plush_nostuffing", /datum/mood_event/plush_nostuffing)
+
+// Jack the Ripper starring plush
+/obj/item/toy/plush/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	if(!grenade)
```

Comment:
Add a check to ensure the item is actually sharp, otherwise playing with plushies is gonna be giving you a negative moodlet for brutally murdering a plush on top of the positive moodlet for playing with plushies